enhancements/OSAC-1382-multi-fabric-east-west-networking/design.md

```
@@ -0,0 +1,487 @@
+
+```python
+class NetworkClassCapabilities(Base):
+    supports_ipv4: bool = False
+    supports_ipv6: bool = False
+    supports_dual_stack: bool = False
+    supports_east_west_ethernet: bool = False
+```
+
+#### Netris API Integration (Validated on zeus12)
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)